resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

```diff
@@ -1,192 +1,263 @@
+			"r_c forgets to help m_c with something.",
+			"r_c blurts out something. m_c wishes that r_c held {PRONOUN/r_c/poss} tongue.",
+			"r_c says something bad about m_c that's been on everyone's mind."
+		],
+		"random_trait_constraint": [
+			"impulsive"
+		]
+	},
+	{
+		"id": "comfort_de_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c bristles after being scolded by r_c.",
+			"r_c laughs at something m_c said that shouldn't have been laughed at.",
+			"m_c sees r_c neglect a Clanmate.",
+			"m_c notices that r_c's smile isn't a real one.",
+			"Every muscle in m_c's body tenses when r_c walks by.",
+			"m_c keeps having bad dreams about r_c.",
+			"m_c always notices r_c lurking nearby.",
+			"m_c badly misinterprets r_c's words.",
+			"r_c keeps pacing around in the clearing late at night and m_c is the only one who notices.",
+			"m_c is discomforted by r_c's muttering.",
+			"m_c feels extremely unsettled when r_c looks at {PRONOUN/m_c/object}.",
+			"m_c can't stop thinking about the way r_c glared at {PRONOUN/r_c/object} Clanmate."
+		]
+	},
+	{
+		"id": "comfort_de_high2",
+		"intensity": "high",
+		"interactions": [
+			"m_c is telling jokes about r_c.",
+			"While m_c is upset over something, r_c stares at {PRONOUN/m_c/object} from afar.",
+			"r_c won't tell m_c what {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} doing.",
+			"The Clan feels uneasy whenever m_c and r_c speak to each other.",
+			"m_c and r_c are joking around together, but the jokes are becoming targeted.",
+			"m_c told r_c to shut up.",
+			"During a conversation with r_c, m_c says something too bluntly.",
+			"m_c calls out r_c for leaving old, rotten prey in the pile.",
+			"r_c thinks m_c is lying when {PRONOUN/r_c/subject} {VERB/r_c/apologize/apologizes}.",
+			"m_c tells r_c that {PRONOUN/r_c/subject} {VERB/r_c/look/looks} terrible today.",
+			"m_c was sarcastic to r_c at an inappropriate time.",
+			"m_c hisses at r_c over something small."
+		],
+		"reaction_random_cat": {
+			"comfortable": "decrease"
+		}
+	},
+	{
+		"id": "comfort_de_high3",
+		"intensity": "high",
+		"interactions": [
+			"m_c shivers after noticing r_c following {PRONOUN/m_c/object} around like a shadow.",
+			"m_c feels a chill after noticing r_c is nearby.",
+			"Watching r_c stalk silently around camp makes m_c feel uneasy.",
+			"m_c doesn't like the way r_c seems to overwhelm the camp with {PRONOUN/r_c/poss} presence.",
+			"Even though r_c is a respectable Clanmate, m_c can't help but think {PRONOUN/r_c/subject} might be up to no good...",
+			"m_c swore {PRONOUN/m_c/subject} saw r_c burying something.",
+			"m_c can smell blood on r_c, but there's nothing there.",
+			"m_c sees a darkness in r_c's eyes.",
+			"A heavy feeling of guilt weights on m_c whenever {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} kind to r_c.",
+			"The bad dreams that m_c has about r_c are progressively getting worse and worse.",
+			"Why does m_c always see r_c out at night?"

```

Comment:
This isn't quite formatted appropriately for a rel event, though I see the intention!  "m_c wonders why they always see r_c out at night." could be a better wording.  Feel free to rework this one or remove it.